pkg/workflow/spec/fnlib/convert.go

```diff
@@ -0,0 +1,49 @@
+package fnlib
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/puppetlabs/nebula-sdk/pkg/workflow/spec/convert"
+	"github.com/puppetlabs/nebula-sdk/pkg/workflow/spec/fn"
+)
+
+var convertMarkdownDescriptor = fn.DescriptorFuncs{
+	DescriptionFunc: func() string { return "Converts a string in markdown format to another applicable syntax" },
+	PositionalInvokerFunc: func(args []interface{}) (fn.Invoker, error) {
+		fn := fn.InvokerFunc(func(ctx context.Context) (m interface{}, err error) {
+			if len(args) != 2 {
```

Comment:
Also, you can put this check outside the invoker. The arity won't change, even though the args might.